.github/workflows/publish_pypi.yml

```diff
@@ -3,31 +3,30 @@ name: Publish package to PyPi
 on:
   push:
     tags:
-      - '*'
+      - "*"
 
 jobs:
+  push:
+    runs-on: ubuntu-latest
+    environment:
+      name: pypi
+      url: https://pypi.org/project/h5p-xblock
+    permissions:
+      id-token: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
 
-   push:
-     runs-on: ubuntu-latest
-     environment:
-       name: pypi
-       url: https://pypi.org/project/h5p-xblock
-     permissions:
-       id-token: write
-     steps:
-       - name: Checkout
-         uses: actions/checkout@v4
-         
-       - name: Set up Python ${{ matrix.python-version }}
-         uses: actions/setup-python@v5
-         with:
-           python-version: '3.12'
+      - name: Set up Python ${{ matrix.python-version }}
```

Comment:
nit: remove ${{ matrix.python-version }} because we are no longer using it now.

Reply:
Sure